### PR TITLE
Do not store passwords user hosting accounts in plain text

### PR DESCRIPTION
**Pull request by jaapmarcus**
It is always a bad idea to do it. It also "breaks" if the user changes the password anyway.

### CODE REVIEW

File: src/modules/Servicehosting/Service.php [hunk 1]
```diff
@@ -123,13 +123,16 @@ public function action_activate(\Model_ClientOrder $order)
...
         $model->username = $username;
         $model->pass = $pass;
-        $this->di['db']->store($model);
 
         if (!isset($c['import']) || !$c['import']) {
             [$adapter, $account] = $this->_getAM($model);
             $adapter->createAccount($account);
         }
 
+        $model->pass = '*******';
```
Comment (BelleNottelling) [inline src/modules/Servicehosting/Service.php:132]: Why is this not up a few lines with the rest of the code related to the model?

Comment (jaapmarcus) [reply src/modules/Servicehosting/Service.php:132]: $model is also used to create the hosting account in 

[$adapter, $account] = $this->_getAM($model);
            $adapter->createAccount($account);

If we set it to ***** and then create the account all user passwords will be the same. 

Ideally we should drop the column for password and forget about it saving it...

Comment (BelleNottelling) [reply src/modules/Servicehosting/Service.php:132]: Dropping the column entirely is an easy enough task

Comment (jaapmarcus) [reply src/modules/Servicehosting/Service.php:132]: Then we should do that imo

**Review by BelleNottelling — APPROVED** [review]
Good enough for now

**Review by John-S4 — APPROVED** [review]

**Pull request merged**